vmdb/db/migrate/20150405141637_remove_port_config_from_container_service.rb

```diff
@@ -0,0 +1,56 @@
+class RemovePortConfigFromContainerService < ActiveRecord::Migration
+  class ContainerService < ActiveRecord::Base
+  end
+
+  class ContainerServicePortConfig < ActiveRecord::Base
+  end
+
+  def up
+    create_table :container_service_port_configs do |t|
+      t.string     :ems_ref
+      t.string     :name
+      t.integer    :port
+      t.string     :target_port
+      t.string     :protocol
+      t.belongs_to :container_service, :type => :bigint
+    end
+
+    say_with_time("Moving container service port records to container service port config table") do
+      ContainerService.all.each do |service|
+        ContainerServicePortConfig.create!(
+            :ems_ref              => "#{service.ems_ref}_#{service.port}_#{service.container_port}_#{service.protocol}",
```

Comment:
what about the protocol field? still part of ems_ref? iirc, there was a discussion to remove it from ems_ref...